core/persistence-jpa/src/main/java/org/apache/syncope/core/persistence/jpa/dao/JPAPlainSchemaDAO.java

```diff
@@ -135,4 +151,17 @@ public void delete(final String key) {
 
         entityManager().remove(schema);
     }
+
+    private <T extends PlainAttr<?>> String getPlainAttrTable(Class<T> plainAttrClass) {
```

Comment:
add `final` to method argument here, or build will fail

Reply:
done